docs/reference/commandline/build.md

```diff
@@ -251,3 +252,22 @@ flag](/reference/run/#specifying-custom-cgroups).
 Using the `--ulimit` option with `docker build` will cause each build step's 
 container to be started using those [`--ulimit`
 flag values](/reference/run/#setting-ulimits-in-a-container).
+
+Currently, you can use `ENV` instructions in a Dockerfile to define variable
```

Comment:
Just noticed this was suggested by @moxiegirl 

@moxiegirl wdyt (this comment, and the comment on adding a header above)?